<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInterfaceClass</class>
 <widget class="QWidget" name="UserInterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserInterface</string>
  </property>
  <property name="windowOpacity">
   <double>500000.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QGraphicsView" name="animation_window">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>CrossCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLabel" name="steps_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>No. of steps</string>
   </property>
  </widget>
  <widget class="QLabel" name="nr_steps_label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="size_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Array size</string>
   </property>
  </widget>
  <widget class="QLabel" name="size_label_nr">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="algorythm_name_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Algorythm</string>
   </property>
  </widget>
  <widget class="QLabel" name="algorythm_name">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="UserInterface.qrc"/>
 </resources>
 <connections/>
</ui>
